docs/midstream-getting-serveless-operator-bugfixes.md

```diff
@@ -0,0 +1,132 @@
+# Getting bugfixes for midstream Serverless Operator
+
+## 1. Introduction
+
+This document explains how to update the midstream Serverless operator image so that you can get bugfixes.
+
+Serverless Operator midstream, unlike the fully released product, does not create patch releases.
```

Comment:
I thought we didnt have patch releases (https://issues.redhat.com/browse/SRVCOM-2131)?
If the goal is continuous delivery for managed services then we could include that to the ticket.
Otherwise who is responsible for the QA of these patches?

Reply:
Sorry I thought we do create patch releases for the product.

Reply:
Let's ignore this thread as we discuss it on slack anyway.